examples/provider/provider_0.12.tf

```diff
@@ -21,7 +21,7 @@ resource "juju_model" "development" {
 resource "juju_application" "wordpress" {
   name = "wordpress"
 
-  model = juju_model.development.name
+  model_uuid = juju_model.development.uuid
```

Comment:
not about this specific line, but what is this entire example about? it's not fixing the provider version to 0.12...